src/main/scala/org/broadinstitute/dsde/workbench/avram/util/RestClient.scala

```diff
@@ -3,15 +3,19 @@ package org.broadinstitute.dsde.workbench.avram.util
 import java.net.URI
 import java.util.logging.Logger
 
-import com.softwaremill.sttp.{HttpURLConnectionBackend, Request, Uri, sttp}
+import cats.effect.IO
+import com.softwaremill.sttp.asynchttpclient.cats.AsyncHttpClientCatsBackend
+import com.softwaremill.sttp.{HttpURLConnectionBackend, Id, Request, SttpBackend, Uri, sttp}
 
 /**
   * Helpful stuff for making rest calls using sttp (https://github.com/softwaremill/sttp).
   */
 trait RestClient {
   private val log: Logger = Logger.getLogger(getClass.getName)
 
-  implicit def sttpBackend = HttpURLConnectionBackend() // TODO: consider refactoring to allow use of SttpBackendStub in tests
+  // Temporarily don't put either of these in implicit scope so that functions can choose one
+  def sttpBackend: SttpBackend[Id, Nothing] = HttpURLConnectionBackend()
+  def catsSttpBackend: SttpBackend[IO, Nothing] = AsyncHttpClientCatsBackend[IO]()
```

Comment:
Question: Why do we need two and why is getUserStatus using  `HttpURLConnectionBackend` while `queryAction`is using `AsyncHttpCatsBackend`?

Reply:
Having 2 is only temporary. I did this to make the change to `IO` only apply to the new code I was adding, which should help keep the PR smaller (doesn't include refactoring existing code) and keep it more compatible with other in-flight PRs. If accepted, I would follow-on quickly with a separate PR for refactoring the existing code away from `HttpURLConnectionBackend`.